models/skipthoughts.py

```diff
@@ -0,0 +1,194 @@
+"""
+A simple model based on skipthoughts sentence embeddings.
+
+To set up:
+    * Execute the "Getting started" wgets in its README
+    * set config['skipthoughts_datadir'] to directory with downloaded files
+    * make skipthoughts.py from https://github.com/ryankiros/skip-thoughts/blob/master/skipthoughts.py 
+        available via import skipthoughts
+
+Inner working: First we compute skipthought embedding of both inputs; then we merge them (multiply & subtract), cancatenate, and compute result (1 MLP layer).
+"""
+
+from __future__ import print_function
+from __future__ import division
+
+
+from keras.models import Graph
+from keras.layers.core import Activation, Dense, Dropout
+from keras.regularizers import l2
+from keras.optimizers import Adam
+
+import pysts.embedding as emb
+import pysts.loader as loader
+import pysts.kerasts.blocks as B
+from pysts.kerasts.objectives import pearsonobj
+
+import numpy as np
+
+
+def config(c):
+    # XXX:
+    c['skipthoughts_datadir'] = "/storage/ostrava1/home/nadvorj1/skip-thoughts/"
+
+    # disable GloVe
+    c['embdim'] = None
+    # disable Keras training
+    c['ptscorer'] = None
```

Comment:
I think doing it this way is a major issue that we'll need to look into. More in a main comment.